src/utils/etl_util.py

```diff
@@ -0,0 +1,44 @@
+def pass_values_of_odm_object_to_orm_object(odm_object, orm_object):
```

Comment:
I don't like generalize this, it is really variable, just convert the data manually

Reply:
done.